src/runtime/audience-action-local-flow.ts

```diff
@@ -100,6 +44,9 @@ export interface AudienceActionLocalParams {
   monetizationFunction?: () => void;
 }
 
+// Default timeout for waiting on ready callback.
```

Comment:
nit: please add the unit in the comment or in the constant name.

Reply:
Done.